### PR TITLE
Improved install script and README file

### PR DESCRIPTION
**Pull request by akkuzinaolga**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,45 +1,52 @@
 #!/bin/bash
-# if you understand this script, then you should install next things manually:
-# - ruby and phantomjs
-# - bundler for install all necessary gems
-# - after all add gems path to $PATH for using bundle and install gems
-# - copy config.yml.example to config.yml and change config.yml as you want
-# - bin/f - is executable file
+# - Installing this terrible app in your Manjaro Linux system.
```
Comment (makkuzin) [inline install.sh:2]: Where comments?

File: README.rdoc [hunk 2]
```diff
@@ -1,6 +1,14 @@
 = forker
 
-Describe your project here
+== Description
 
-:include:forker.rdoc
+App for search of forks in live. Currently works only with tennis events.
 
+== Install
```
Comment (makkuzin) [inline README.rdoc:7]: Is this comments from install script?

**Comment by makkuzin** [comment]
Are you sure?

File: README.rdoc (hunk 2 above)
Comment (akkuzinaolga) [inline README.rdoc:7]: Yes it is

**Pull request merged**